conf4j-core/src/main/java/com/sabre/oss/conf4j/internal/factory/jdkproxy/AbstractJdkProxyConfigurationInvocationHandler.java

```diff
@@ -46,7 +50,7 @@
 
 abstract class AbstractJdkProxyConfigurationInvocationHandler implements InvocationHandler, Serializable, ConfigurationPropertiesAccessor {
     private static final long serialVersionUID = 1L;
-
+    private transient Constructor<MethodHandles.Lookup> constructor;
```

Comment:
This field should be static - lookup constructor should be done only once. You can reuse _generateConstructor()_ method (I would call it _getLookupConstructor()_):

```java
    private static final Constructor<Lookup> lookupConstructor = getLookupConstructor();
    
    ...
    
    private static Constructor<Lookup> getLookupConstructor() {
        try {
            Constructor<Lookup> constructor = Lookup.class.getDeclaredConstructor(Class.class, int.class);
            constructor.setAccessible(true);
            return constructor;
        } catch (NoSuchMethodException e) {
            throw new RuntimeException(e);
        }
    }
```

In conf4j, we put all private methods at the end of class - please follow this convention.